src/content_script/mastodonInject.js

```diff
@@ -103,46 +101,69 @@ async function injectFollowButton() {
  */
 async function injectInteractionButtons() {
     const INJECTED_REPLY_CLASS = "mastodon-simplified-federation-injected-interaction";
-    const replyButtons = await waitForElement(
-        "#mastodon .item-list[role='feed'] article[data-id] .status__action-bar button," +
-        "#mastodon .detailed-status__wrapper .detailed-status__action-bar button",
-        true,
-        TIMEOUT_DURATION,
-    );
-    replyButtons.forEach((button) => {
-        try {
-            if (!button.classList.contains(INJECTED_REPLY_CLASS)){
-                button.addEventListener("click", onClickInteract);
-                button.classList.add(INJECTED_REPLY_CLASS);
+    try {
+        const replyButtons = await waitForElement(
+            "#mastodon .item-list[role='feed'] article[data-id] .status__action-bar button," + // timeline / user profile
+            "#mastodon .detailed-status__wrapper .detailed-status__action-bar button," + // status with no replies
+            "#mastodon .status__wrapper .status__action-bar button", // status with replies
```

Comment:
Clean code would say put this into three variables, instead of comments at the end and concatenate these (as you do anyway), but I'd be fine with this, too.